test/item_repository_test.rb

```diff
@@ -68,14 +68,14 @@ def test_find_by_merchant_id
   end
 
   def test_find_by_created_at
-    output = item_repository.find_by_created_at(DateTime.parse('2012-03-27 14:53:59 UTC')).id
+    output = item_repository.find_by_created_at(Date.parse('2012-03-27 14:53:59 UTC')).id
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.